lib/tether/toolbox.go

```diff
@@ -139,7 +139,9 @@ func (t *Toolbox) Reload(config *ExecutorConfig) error {
 	t.authIDs[config.ID] = struct{}{}
 	// we also allow any device IDs that are attached
 	for _, mspec := range config.Mounts {
-		t.authIDs[mspec.Source.Host] = struct{}{}
+		// mounstpect.source.path is the disk label for vmdks
+		// XXX: this is not the case for other volumes, eg nfs vols.
```

Comment:
the `mspec.Source.Scheme` should be `label` if I recall correctly for vmdk mounts and can be used as a check.